### PR TITLE
Clarify "Top level ORs"

### PR DESCRIPTION
**Pull request by DavidGrashton**
Adding an example to clarify what "Top level ORs" are, in response to some customer confusion.

### Merge instructions

- [√] Please merge after reviewing

### CODE REVIEW

File: content/en/monitors/downtimes/_index.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ There are a few limitations that are **not supported** which include:
 
 * More than two levels of nesting, such as `team:app AND (service:auth OR (service:graphics-writer AND (env:prod OR (type:metric AND status:ok))))`, are not supported. At most, Downtimes accept two levels of nesting. Use separate Downtimes instead to break down the logic.
 * Negation is only supported for key/value pairs and tags with `OR`. For example, `-key:value` and `-key(A OR B)`. Scopes such as `-service:(A AND B)`, `service:(-A OR -B)`, or `-service(A B)` are not supported.
-* Top level ORs are not supported, for example, `service:A OR host:X`. This requires two separate Downtimes.
+* Top level ORs are not supported. For example, `service:A OR service:B` will work but `service:A OR host:X` will not; this requires two separate Downtimes.
```
Comment (urseberry) [inline content/en/monitors/downtimes/_index.md:89]:
```suggestion
* Top level ORs are not supported. For example, `service:A OR service:B` is valid, but `service:A OR host:X` does not work. An OR between two different top level tags requires two separate Downtimes.
```

**Review by urseberry — CHANGES_REQUESTED** [review]
Thank you for improving the public documentation!

File: content/en/monitors/downtimes/_index.md (hunk 1 above)
Comment (DavidGrashton) [reply content/en/monitors/downtimes/_index.md:89]: Thank you for improving my improvement! This is much clearer

**Review by michaelcretzman — APPROVED** [review]
Approved with one edit

File: content/en/monitors/downtimes/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DavidGrashton) [reply content/en/monitors/downtimes/_index.md:89]: Hi Michael, that is definitely still correct, but can we use Ursula's version? I had intended to accept this suggestion but I must have pressed the wrong thing.

"Top level ORs are not supported. For example, `service:A OR service:B` is valid, but `service:A OR host:X` does not work. An OR between two different top level tags requires two separate Downtimes."

**Review by DavidGrashton — COMMENTED** [review]
LGTM!

**Pull request merged**